<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoWindow</class>
 <widget class="QWidget" name="InfoWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Изменение информации</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>timetable_icon.png</normaloff>timetable_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(74, 112, 139);</string>
  </property>
  <widget class="QPushButton" name="pushButton_back">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(108, 166, 205);
border: 2px solid gray;
border-radius: 10px;
padding: 3 8px;</string>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>250</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(108, 166, 205);
border: 2px solid gray;
border-radius: 10px;
padding: 3 8px;
</string>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lession">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Название предмета</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(108, 166, 205);
border: 2px solid gray;
border-radius: 10px;
padding: 3 8px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_session">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Наличие сессии</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_notable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>171</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Проверка посещаемости</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_lecture">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Лекция</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_dopsa">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Вероятность допсы</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_session">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_notable">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_lection">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>301</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_profile">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Профильность</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_profile">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>110</y>
     <width>87</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_session_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>161</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Оценка преподавателя</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_prepod">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>80</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox_dopsa">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>200</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
